src/Uno.UI/Microsoft/UI/Xaml/Controls/PersonPicture/PersonPicture.cs

```diff
@@ -203,23 +203,33 @@ void UpdateIfReady()
 
 		var templateSettings = TemplateSettings;
 		templateSettings.ActualInitials = initials;
-		if (imageSrc != null)
+
+		if (imageSrc is not null)
```

Comment:
Are the changes in this method refactoring only? It may be better to keep this closer to the original code from WinUI to ease future revisions of the port.

Reply:
Good point. Will revert back